test/routes/setup_test.js

```diff
@@ -0,0 +1,18 @@
+let path = require('path');
```

Comment:
Mocha does have support for `setup` and `teardown` blocks outside of a suite that get applied to all suites